src/DIRAC/WorkloadManagementSystem/Client/DownloadInputData.py

```diff
@@ -227,9 +228,9 @@ def __checkDiskSpace(self, totalSize):
         """
         diskSpace = getDiskSpace(self.__getDownloadDir(False))  # MB
         availableBytes = diskSpace * 1024 * 1024  # bytes
-        # below can be a configuration option sent via the job wrapper in the future
-        # Moved from 3 to 5 GB (PhC 130822) for standard output file
-        bufferGBs = 5.0
+        bufferGBs = gConfig.getValue(
+            os.path.join(getSystemSection("WorkloadManagement/JobWrapper"), "JobWrapper", "InputDataBufferGBs"), 5.0
```

Comment:
This should be documented in the release notes and other places?

Reply:
I feel like there might be a better name than `InputDataBufferGBs` is this is a new option. How about `MinScratchSpaceGB`?

Reply:
Changed